src/client/components/characters/CharacterEditor.tsx

```diff
@@ -96,7 +96,7 @@ export const conditionIcons = {
   hasted: speedometer,
   cursed: voodooDoll,
   "hunters mark": bullseye,
-  polymorphed: snail,
+  polyMorphed: snail,
```

Comment:
This change seems a bit weird, since the polymorph spell is also called "polymorph" and not "poly morph"

Reply:
Hmm, that's fair. Maybe we should consider renaming it to something more general (e.g. transfigured), so that it is not specific to one particular spell?

Reply:
Hmm, I see where you are going with this and I agree that _transfiguration_ is a more general term. Another word that came to my mind is _shapeshifting_.
I wonder what term (_shapeshifting_, _polymorphed_, _transfigured_) we, but also other people, tend to generally use. I don't really know. 🤔

Reply:
I undid the rename for now. I think we might need to talk about our conditions at some point anyway. I mean the snail icon for polymorph was a joke for this particular campaign, so it might not be the best way to represent this condition.